ETL/EmployeeObj.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+
+namespace ETL
+{
+  public class EmpObj
+  {
+    public int empNum;
```

Comment:
naming convention says that everything is PascalCase except for parameters which are camelCase